actix-tls/Cargo.toml

```diff
@@ -98,9 +103,12 @@ tokio-rustls-024 = { package = "tokio-rustls", version = "0.24", optional = true
 webpki-roots-025 = { package = "webpki-roots", version = "0.25", optional = true }
 
 # rustls v0.22
-rustls-pki-types-1 = { package = "rustls-pki-types", version = "1", optional = true }
+rustls-pki-types-1 = { package = "rustls-pki-types", version = "1", optional = true } # Also used for rustls v0.23
```

Comment:
Should these deps still be under the 0.22 comment since they're now used by both 0.22 and 0.23?

Reply:
dw about it, i'll sort this out before release